docs/source/design.rst

```diff
@@ -61,6 +61,40 @@ You can get the list of supported modules for a given device instance using :att
     If you only need some module-specific information,
     you can call the wanted method on the module to avoid using :meth:`~kasa.SmartDevice.update`.
 
+Protocols and Transports
+************************
+
+The library supports two different TP-Link protocols, ``IOT`` and ``SMART``.
+``IOT`` is the original Kasa protocol and ``SMART`` is the newer protocol supported by TAPO devices and newer KASA devices.
```

Comment:
Probably worth mentioning here, that the protocols imply different set of commands? In the future, we can improve the module docs on that, too.

Reply:
Done